stdlib/3.6/secrets.pyi

```diff
@@ -0,0 +1,13 @@
+# Stubs for secrets (Python 3.6)
+
+from typing import Any, Callable, Optional
+from hmac import compare_digest as compare_digest
+from random import SystemRandom as SystemRandom
+
+randbits = ...  # type: Callable[[int], int]
+choice = ...  # type: Callable[..., Any]
```

Comment:
Change those to be regular defs, copy from random.pyi:Random (removing `self` obviously). Then you'll be able to express `choice` correctly as:
```py3
_T = TypeVar('_T')
def choice(seq: Sequence[_T]) -> _T: ...
```